packages/fxa-shared/connected-services/factories.ts

```diff
@@ -241,11 +241,11 @@ export class ConnectedServicesFactory {
       client.clientId = oauthClient.client_id;
       client.scope = oauthClient.scope;
       client.createdTime = Math.min(
-        client.createdTime || Number.MAX_VALUE,
+        client.createdTime || Number.POSITIVE_INFINITY,
         oauthClient.created_time
       );
       client.lastAccessTime = Math.max(
-        client.lastAccessTime || Number.MIN_VALUE,
+        client.lastAccessTime || 0,
```

Comment:
Why bother changing these?

Reply:
Originally, before ported to typescript, there was no type check that lastAccessTime could be a number. Now typescript enforces this. As a result I had to add fallback values. 

Sense there is some suspicion around last access time, this is some code that caught my attention. For time, I think 0 seems like the most appropriate minimum. And for the maximum positive infinity seems more correct (ie Number.POSITIVE_INFINITY > Number.MAX_VALUE is true). IDK, probably just overthinking this...